packages/chart/src/Pie.ts

```diff
@@ -220,14 +261,100 @@ export class Pie extends SVGWidget {
+            yShift -= heightDiff / 2;
+        } else if (top < 0 && absTop > gY) {
+            yShift = absTop - gY + (this.labelHeight() / 2);
+            yShift += heightDiff / 2;
+        } else {
+            if (absTop > bottom) {
+                yShift += (absTop - bottom) / 2;
+            } else {
+                yShift -= (bottom - absTop) / 2;
+            }
+        }
+        const pos = this.pos();
+        this.pos({
+            y: pos.y + yShift,
+            x: pos.x
+        });
+    }
+    adjustForOverlap() {
+        const labelHeight = this.labelHeight();
+        this._quadIdxArr.forEach((arr, quad) => {
+            this._quadIdxArr[quad].sort((a, b) => {
+                if (quad === 1 || quad === 2) {
+                    return this._labelPositions[a].top > this._labelPositions[b].top ? -1 : 1;
+                } else if (quad === 0 || quad === 3) {
+                    return this._labelPositions[a].top > this._labelPositions[b].top ? 1 : -1;
+                }
+            });
+            let prevTop;
+            this._quadIdxArr[quad].forEach((n, i) => {
+                if (i > 0) {
+                    if (quad === 1 || quad === 2) {
+                        if (prevTop < this._labelPositions[n].bottom) {
+                            const overlap = this._labelPositions[n].bottom - prevTop;
+                            this._labelPositions[n].top -= overlap;
+                            this._labelPositions[n].bottom -= overlap;
+                        }
+                    } else if (quad === 0 || quad === 3) {
+                        if (prevTop + labelHeight > this._labelPositions[n].top) {
+                            const overlap = Math.abs(this._labelPositions[n].top) - Math.abs(prevTop + labelHeight);
+                            this._labelPositions[n].top -= overlap;
+                            this._labelPositions[n].bottom -= overlap;
+                        }
+                    }
+                }
+                prevTop = this._labelPositions[n].top;
+            });
+        });
+        this._minLabelTop = 0;
+        this._maxLabelBottom = 0;
+        this._quadIdxArr.forEach((arr, quad) => {
+            this._quadIdxArr[quad].forEach((n, i) => {
+                if (this._minLabelTop > this._labelPositions[n].top) {
+                    this._minLabelTop = this._labelPositions[n].top;
+                }
+                if (this._maxLabelBottom < this._labelPositions[n].bottom) {
+                    this._maxLabelBottom = this._labelPositions[n].bottom;
+                }
+            });
+        });
+    }
     exit(_domNode, _element) {
         SVGWidget.prototype.exit.apply(this, arguments);
     }
 
     updateD3Pie() {
+        const startAngle = normalizeRadians(degreesToRadians(this.startAngle() - 180));
         this.d3Pie
             .padAngle(0.0025)
+            .startAngle(-Math.PI + startAngle)
+            .endAngle(Math.PI + startAngle)
```

Comment:
Can you remove the subtraction of 180 degrees and simply change the use of PI here?
(180 degrees === Math.PI in radians)